e2e/cypress/integration/api/iam/policies_api.spec.ts

```diff
@@ -26,6 +26,13 @@ const chefManagedPolicyIDs = [
   'ingest-access'
 ];
```

Comment:
Suggested cleanup just above here--get rid of duplication (and magic strings):
```suggestion
const chefManagedPolicyIDs = [
  constants.AdminPolicyID,
  constants.EditorPolicyID,
  constants.ViewerPolicyID,
  constants.IngestPolicyID
];
```

Reply:
We don't have constants for these in the UI world (though maybe we should) so leaving this and adding a comment.